src/drive/web/modules/drive/Toolbar/components/AddButton.jsx

```diff
@@ -8,13 +8,19 @@ import { AddMenuContext } from 'drive/web/modules/drive/AddMenu/AddMenuProvider'
 
 export const AddButton = () => {
   const { t } = useI18n()
-  const { anchorRef, handleToggle, isDisabled } = useContext(AddMenuContext)
+  const {
+    anchorRef,
+    handleToggle,
+    isDisabled,
+    handleOfflineClick,
+    isOffline
+  } = useContext(AddMenuContext)
 
   return (
-    <div ref={anchorRef}>
+    <div ref={anchorRef} {...isOffline && { onClick: handleOfflineClick }}>
```

Comment:
nit: I find it more readable onClick={isOffline && handleOfflineClick}

Reply:
je suis d'accord, je corrige